config/prod/sjcg-rnaseq-wf.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: sjcg-rnaseq-wf
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  # OpenPorts: ["80","443"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "NF-OSI"
+  # The resource owner
+  OwnerEmail: "sara.gosline@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "ami-0b898040803850657"      # AWS linux 2 AMI 2.0.20190618 x86_64 HVM gp2
+  # AMIId: "ami-0a313d6098716f372"      # Ubuntu, 18.04 LTS, amd64 bionic image build on 2019-02-12
```

Comment:
Uncomment the top one and use this line

Reply:
i dont understand what this means.

Reply:
you chose a ubuntu distro (on line 5) the comment says you need to choose an AMIId that matches ubuntu.  you need to comment out the aws linux and uncomment the ubuntu AMI.

Reply:
got it.